src/extensions/extensionInterface.ts

```diff
@@ -196,10 +196,18 @@ export interface Plugins {
     sortingUnitMetrics: {[key: string]: SortingUnitMetricPlugin}
 }
 
-export interface SortingViewProps {
+interface ViewProps {
+    plugins: Plugins
+    hither: HitherContext
+    calculationPool: CalculationPool
+    width?: number
```

Comment:
width and height are now optionally passed to the plugin views. Then the plugin views have the choice of whether or not to respect these.